lib/load_brocfile.js

```diff
@@ -4,23 +4,71 @@ const path = require('path');
 const findup = require('findup-sync');
 const esmRequire = require('esm')(module);
 
+/**
+ * Require a brocfile via either ESM or TypeScript
+ *
+ * @param {String} brocfilePath The path to the brocfile
+ * @returns {*}
+ */
+function requireBrocfile(brocfilePath) {
+  let brocfile;
+
+  if (brocfilePath.match(/\.ts$/)) {
+    const configPath = findup('tsconfig.json', {
```

Comment:
Why is this needed at all?

https://github.com/TypeStrong/ts-node#loading-tsconfigjson suggests that it will find the tsconfig.json just fine on its own, no?

Reply:
@oligriffiths thoughts?

Reply:
Well, the point of this is to ensure that we either use the tsconfig or we use the defaults, but not both. node-ts overwrites the tsconfig options with the passed in config https://github.com/TypeStrong/ts-node/blob/2609663bf3a5b14d76c0dfe989ceba67e68d5efa/src/index.ts#L486

Reply:
Removed